package.json

```diff
@@ -94,8 +97,9 @@
   "scripts": {
     "postinstall": "./tools/task-runner/runner githooks --verbose",
     "sass-watch": "node-sass -w ./static/src/stylesheets -o ./static/target/stylesheets --source-map=true",
-    "css-watch": "gulp --cwd ./dev watch:css",
+    "babel-watch": "BABEL_ENV=production babel ./static/src/javascripts --watch -d ./static/transpiled/javascripts --ignore .eslintrc.js >/dev/null 2>&1",
```

Comment:
Ah. Well, I suppose at least that's clear, as they'll probably diverge at some point. Could always have a `common` `rc` and then specific env ones that extend (or concat arrays etc) from common? Not for now though but may be a pattern that works for these configs if they get too antisocial.